frontend/components/PerceivedDemographics.jsx

```diff
@@ -75,18 +92,20 @@ class PerceivedDemographicsContainer extends Component {
             </select>
           </label>
 
-          <input type="submit" value="Submit" />
+          <button type="submit">Submit</button>
         </form>
-        <CarouselContainer/>
+        <CarouselContainer />
       </div>
-    )
+    );
   }
 }
 
-const mapStateToProps = state => ({});
+PerceivedDemographicsContainer.propTypes = {
+  onEnter: PropTypes.func.isRequired,
+};
 
 const mapDispatchToProps = dispatch => ({
   onEnter: () => dispatch(requestFaces()),
 });
 
-export default connect(mapStateToProps, mapDispatchToProps)(PerceivedDemographicsContainer);
+export default connect(null, mapDispatchToProps)(PerceivedDemographicsContainer);
```

Comment:
mapStateToProps was returning an empty object, so passing a falsy value here lets us skip having to declare the method at all.